.ebextensions/01_logs_streamtocloudwatch.config

```diff
@@ -0,0 +1,76 @@
+###################################################################################################
+#### The following file installs and configures the AWS CloudWatch Logs agent to push logs to a Log
+#### Group in CloudWatch Logs.
+#### 
+#### The configuration below sets the logs to be pushed, the Log Group name to push the logs to and
+#### the Log Stream name as the instance id. The following files are examples of logs that will be
+#### streamed to CloudWatch Logs in near real time:
+#### 
+#### /var/app/current/storage/logs/laravel.log
+#### 
+#### You can then access the CloudWatch Logs by accessing the AWS CloudWatch Console and clicking
+#### the "Logs" link on the left. The Log Group name will follow this format:
+####
+#### /aws/elasticbeanstalk/<environment name>/<full log name path>
+####
+#### Please note this configuration can be used additionally to the "Log Streaming" feature:
+#### http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/AWSHowTo.cloudwatchlogs.html
+####
+#### Example configuration file:
+#### https://github.com/awsdocs/elastic-beanstalk-samples/blob/master/configuration-files/aws-provided/instance-configuration/logs-streamtocloudwatch-linux.config
+###################################################################################################
+
+option_settings:
+  - namespace: aws:elasticbeanstalk:cloudwatch:logs
+    option_name: StreamLogs
+    value: true
+  - namespace: aws:elasticbeanstalk:cloudwatch:logs
+    option_name: DeleteOnTerminate
+    value: false
+  - namespace: aws:elasticbeanstalk:cloudwatch:logs
+    option_name: RetentionInDays
+    value: 7
+
+packages:
+  yum:
+    awslogs: []
+
+files:
+  "/etc/awslogs/awscli.conf" :
+    mode: "000600"
+    owner: root
+    group: root
+    content: |
+      [plugins]
+      cwlogs = cwlogs
+      [default]
+      region = `{"Ref":"AWS::Region"}`
+
+  "/etc/awslogs/awslogs.conf" :
+    mode: "000600"
+    owner: root
+    group: root
+    content: |
+      [general]
+      state_file = /var/lib/awslogs/agent-state
+
+  "/etc/awslogs/config/logs.conf" :
+    mode: "000600"
+    owner: root
+    group: root
+    content: |
+      [/var/app/current/storage/logs]
+      datetime_format = %Y-%m-%d %H:%M:%S
+      log_group_name = `{"Fn::Join":["/", ["/aws/elasticbeanstalk", { "Ref":"AWSEBEnvironmentName" }, "var/app/current/storage/logs"]]}`
+      log_stream_name = logs
```

Comment:
I'm seeing that [the original example](https://github.com/awsdocs/elastic-beanstalk-samples/blob/main/configuration-files/aws-provided/instance-configuration/logs-streamtocloudwatch-linux.config#L69) stated an `{instance_id}` for `log_stream_name`.
I wonder if setting the `log_stream_name` to any arbitrary name is going to mess up with multi-environment apps?

Reply:
depends if you want log streams per instance or all in one stream.

Reply:
![image](https://user-images.githubusercontent.com/10818702/116664366-a5664c00-a998-11eb-82a6-7acdf51e1752.png)

Reply:
http://serebrov.github.io/html/2015-06-17-cloudwatch-logs-single-stream.html

Reply:
probably if you have use eb to host many laravel instances it would be smarter to group them on instance id

Reply:
I have added the {instance_id} instead to be easier to track down the logs by instance.